llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3610,15 +3621,10 @@ def int_aarch64_sve_extq : AdvSIMD_2VectorArgIndexed_Intrinsic;
 //
 // SVE2.1 - Move predicate to/from vector
 //
-def int_aarch64_sve_pmov_to_pred_lane :
-    DefaultAttrsIntrinsic<[LLVMScalarOrSameVectorWidth<0, llvm_i1_ty>],
-                          [llvm_anyvector_ty, llvm_i32_ty],
-                          [IntrNoMem, ImmArg<ArgIndex<1>>]>;
```

Comment:
Can we  add `let TargetPrefix = "aarch64" in { and` still keep the definitions here?
As they do not repeat and we have all the definitions for pmov

Reply:
We could, but will it not be better to have split like that, in case when in the future another intrinsic will have same pattern to this one ?

Reply:
Yeah, I am not sure what is better. 
The only problem is that we have  the definition for the pmov scattered, some here and some there. 
It would be nice to have a pattern, either way is fine.

Reply:
Could you please clarify what do you mean ? I am not sure I follow. To me it seems like all intrinsics in this file are split like this. If you are referring to int_aarch64_sve_pmov_to_vector_lane_merging I don't think that one actually works as well, as it will have the same issue as pmov_to_pred had before this patch.

Reply:
Made pattern consistent by applying it to pmov_to_vector as well